polyfill/README.md

```diff
@@ -1,55 +1,41 @@
-# Temporal Polyfill
+# Non-Production, Test-Only Temporal Polyfill
 
-**Polyfill for [Proposal: Temporal](https://github.com/tc39/proposal-temporal)**
+A non-production polyfill was built to validate this proposal, and continues in this repo for the sole purpose of running tests.
 
-**NOTE: We encourage you to experiment with the polyfill, but don't use it in production!**
-**The API may change based on feedback from implementers, and the current non-production polyfill is very slow for some operations.**
-Please report bugs in the [issue tracker](https://github.com/tc39/proposal-temporal/issues).
+**DO NOT use this polyfill in your own projects! Instead, please use one of the polyfills listed in the table [here](../#polyfills).**
 
-Please run the polyfill with Node.js 12 or later.
+When viewing the reference documentation, this non-production polyfill is automatically loaded in your browser.
+You can experiment with Temporal by opening your browser's developer tools console.
 
-## Documentation
-
-Reference documentation and examples can be found [here](https://tc39.es/proposal-temporal/docs/index.html).
-
-A cookbook to help you get started and learn the ins and outs of Temporal is available [here](https://tc39.es/proposal-temporal/docs/index.html)
+Please report polyfill bugs in the [issue tracker](https://github.com/tc39/proposal-temporal/issues).
 
-## Import as a Module
+The polyfill requires Node.js 12 or later.
 
-You can depend on the unstable Temporal polyfill in your personal projects:
-
-```bash
-$ npm install --save proposal-temporal
-```
-
-In code:
-
-```javascript
-const { Temporal } = require('proposal-temporal');
-```
+## Documentation
 
-Or, import the polyfill as an ES6 module:
+Reference documentation and examples can be found [here](https://tc39.es/proposal-temporal/docs/index.html).
 
-```javascript
-import { Temporal } from 'proposal-temporal/lib/index.mjs';
-```
+A [cookbook](https://tc39.es/proposal-temporal/docs/index.html) of common use cases can help you get started with Temporal.
 
 ## Node REPL with Temporal
```

Comment:
We might consider leaving out this whole section, and just pointing people to the documentation playground. I'm fine with keeping it, personally, but maybe for all avoidance of doubt about the status?

Reply:
Whoops, missed this comment the first time. Sounds good, I'll remove.